rancher/rke2
# Bump k3s and related modules

#### Proposed Changes ####

* Update k3s and related modules: https://github.com/k3s-io/k3s/compare/2561285e8e99...d64a8da87162
* Update runc to track with k3s
* Update wharfie code to track with API changes

#### Types of Changes ####

module update

#### Verification ####

* See linked issues
* Check `runc --version`

#### Linked Issues ####

* https://github.com/rancher/rke2/issues/2159
* Additional fixes for https://github.com/rancher/rke2/issues/570:
    https://github.com/k3s-io/k3s/pull/4700
* https://github.com/k3s-io/k3s/issues/4520
* https://github.com/k3s-io/k3s/issues/4674
* https://github.com/k3s-io/k3s/issues/4713

#### Further Comments ####
